cryptofeed/exchange/ftx.py

```diff
@@ -332,6 +365,33 @@ async def _fill(self, msg: dict, timestamp: float):
                             timestamp=float(timestamp_normalize(self.id, fill['time'])),
                             receipt_timestamp=timestamp)
 
+    async def _index_price(self, pairs: list):
+        # Continously poll list of tickers from the REST API to get most recent index price
+        last_update = {}
+
+        while True:
+            for index_name in pairs:
+                derivative = self._index_to_derivative(index_name)
+                if not derivative:
+                    continue
+                end_point = f"{self.api}/futures/{derivative}"
+                data = await self.http_conn.read(end_point)
+                data = json.loads(data, parse_float=Decimal)
+                timestamp = time()
```

Comment:
is there anything from the endpoint you can use for the exchange_timestamp? this will be difficult to dedupe

Reply:
I guess this is related to the new architecture?
```
{"success":true,"result":{"name":"1INCH-PERP","underlying":"1INCH","description":"1INCH Token Perpetual Futures","type":"perpetual","expiry":null,"perpetual":true,"expired":false,"enabled":true,"postOnly":false,"priceIncrement":0.0001,"sizeIncrement":1.0,"last":2.4915,"bid":2.4896,"ask":2.4919,"index":2.49259721659025,"mark":2.4941,"imfFactor":0.0005,"lowerBound":2.3651,"upperBound":2.6172,"underlyingDescription":"1INCH Token","expiryDescription":"Perpetual","moveStart":null,"marginPrice":2.4941,"positionLimitWeight":20.0,"group":"perpetual","change1h":-0.0789201565846813,"change24h":-0.05105962028687745,"changeBod":-0.11600623803785355,"volumeUsd24h":20605279.8868,"volume":7634664.0,"openInterest":1442881.0,"openInterestUsd":3598689.5021}}
```

Unfortunately there is no timestamp, and no other obvious field to establish total ordering between different clients